uktrade/data-hub-frontend
# Format postcodes in export CSV request

## Description of change

Bug: Currently when you filter companies by multiple postcodes and download the results, no results display in the CSV file.

The bug was due to the postcodes not being formatted into an array in the request body to the API. 

This PR adds the existing middleware, which formats postcodes the user has filtered by, to the list of middleware run when a GET request is made to the export route (`urls.companies.export.route`). I have also renamed this middleware file to be clearer.

There are currently no tests for the download feature, so I am adding a ticket for some e2e testing for this in the tech debt backlog.

## Test instructions

- Filter by 2 or more postcodes on the companies collection page
- Click download (the filters will need to return some results in order to download)
- Check that the download has results

## Screenshots
### Before
![Screenshot 2020-06-22 at 13 08 35](https://user-images.githubusercontent.com/22460823/85285830-7e421280-b489-11ea-9d7a-f6f26574fea2.png)

### After
![Screenshot 2020-06-22 at 13 07 41](https://user-images.githubusercontent.com/22460823/85285757-5e125380-b489-11ea-994d-5c77947b53d8.png)


## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
